src/lamp_py/tableau/pipeline.py

```diff
@@ -58,6 +61,8 @@ def start_hyper_updates() -> None:
         HyperRtAlerts(),
         HyperBusPerformanceAll(),
         HyperBusPerformanceRecent(),
+        HyperGlidesTripUpdates(),
+        HyperGlidesOperatorSignIns(),
```

Comment:
These should also be added to the `parquet_update_jobs` list so that the tableau parquet files get created.